js/common/lib/tensor-impl.ts

```diff
@@ -196,17 +196,31 @@ export class Tensor implements TensorInterface {
     const {height, width} = options;
 
     const norm = options.norm;
```

Comment:
There are multiple places to check whether `norm` is `undefined` in the code below.

There are 2 ways to optimize this:

1) use `const norm = options.norm ?? {};` instead. This makes sure that `norm` is not `undefined`

2) in the check below, use `norm?.mean === undefined` instead of `norm === undefined || norm.mean === undefined`.